CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Before releasing:
 ## [Unreleased]
 
 ### Added
+feedforward.rs
```

Comment:
This isn't how the changelog is supposed to be done, look at the other versions for how you should write this.

Reply:
I'd recommend checking out the CONTRIBUTING.md file for more info